cardano-node/src/Cardano/CLI/Tx/Generation.hs

```diff
@@ -816,16 +770,22 @@ createMoreFundCoins llTracer
                         }
       -- Create and sign splitting txs.
       splittingTxs  = createSplittingTxs pInfoConfig
-                                             (txDetails fvs, sourceKey)
-                                             numSplittingTxOuts
-                                             numOutsPerSplittingTx
-                                             42
-                                             txOut
-                                             []
-  liftIO $ forM_ splittingTxs $ \(tx, txDetailsList) -> do
+                                         (genesisTxDetails, sourceKey)
+                                         numSplittingTxOuts
+                                         numOutsPerSplittingTx
+                                         42
+                                         txOut
+                                         []
+  -- Submit all splitting transactions sequentially.
+  liftIO $ forM_ splittingTxs $ \(tx, _) ->
     submitTx socketFp pInfoConfig (CoreId 0) tx llTracer
```

Comment:
So all the (splitting) transactions go into a single node (`CoreId 0`)?

Reply:
Yes.

Reply:
this is not subject to benchmarking; it's OK